src/main/java/com/worldwarofants/game/arch/console/ConsoleOutputHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.worldwarofants.game.arch.console;
+
+import java.util.concurrent.TimeUnit;
+
+public class ConsoleOutputHandler{
+
```

Comment:
Currently it wouldn't break anything to make instances of this, but just to be sure and to save people the confusion from trying, you should make a private empty constructor for this class. Also add a comment that it's on purpose. 

That is done for similar classes like this one, which only have static methods, so you can disable the ability for instances to be made of it.

Reply:
Updated to include empty constructor